code/drasil-example/projectile/lib/Drasil/Projectile/Body.hs

```diff
@@ -165,14 +174,69 @@ refDB = rdb citations concIns
 concIns :: [ConceptInstance]
 concIns = assumptions ++ funcReqs ++ goals ++ nonfuncReqs
 
--------------------------
+----------------------------------------
 -- Characteristics of Intended Reader --
--------------------------
+----------------------------------------
 
 charsOfReader :: [Sentence]
 charsOfReader = [phrase undergraduate +:+ S "level 1" +:+ phrase Doc.physics,
                  phrase undergraduate +:+ S "level 1" +:+ phrase calculus]
 
+-----------------
+--SystemContext--
+-----------------
+
+sysCtxIntro :: Contents
+sysCtxIntro = foldlSP
+  [refS sysCtxFig1, S "shows the" +:+. phrase sysCont,
+   S "A circle represents an entity external to the", phrase software
+   `sC` phraseNP (the user), S "in this case. A rectangle represents the",
+   phrase softwareSys, S "itself", sParen (short projectileTitle) +:+. EmptyS,
+   S "Arrows are used to show the data flow between the", phraseNP (system
+   `andIts` environment)]
+
+sysCtxDesc :: Contents
+sysCtxDesc = foldlSPCol [S "The interaction between the", phraseNP (product_
+   `andThe` user), S "is through an application programming" +:+.
+   phrase interface, S "The responsibilities of the", phraseNP (user 
+   `andThe` system), S "are as follows"]
+
+sysCtxUsrResp :: [Sentence]
+sysCtxUsrResp = [S "Provide initial" +:+ pluralNP (condition `ofThePS`
+  physical) +:+ S "state of the" +:+ phrase motion +:+ S "and the" +:+ plural inDatum +:+ S "related to the" +:+
+  phrase projectileTitle `sC` S "ensuring no errors in the" +:+
+  plural datum +:+. S "entry",
+  S "Ensure that consistent units are used for" +:+. pluralNP (combineNINI input_ Doc.variable),
+  S "Ensure required" +:+
+  namedRef (SRS.assumpt ([]::[Contents]) ([]::[Section])) (phrase software +:+ plural assumption) +:+
+  S "are appropriate for any particular" +:+
+  phrase problem +:+ S "input to the" +:+. phrase software]
+
+sysCtxSysResp :: [Sentence]
+sysCtxSysResp = [S "Detect data type mismatch, such as a string of characters" +:+
+  phrase input_ +:+. S "instead of a floating point number",
+  S "Determine if the" +:+ plural input_ +:+ S "satisfy the required" +:+.
+  pluralNP (physical `and_` softwareConstraint),
+  S "Calculate the required" +:+. plural output_]
+
+sysCtxResp :: [Sentence]
+sysCtxResp = [titleize user +:+ S "Responsibilities",
+  short projectileTitle +:+ S "Responsibilities"]
```

Comment:
```suggestion
sysCtxResp = map (\x -> titleize x +:+ S "Responsibilities") [user, projectileTitle]
```
Of course, if `titleize projectileTitle` won't work for whatever reason, this will have to be modified.

Reply:
titleize will not work on projectileTitle, Is this ok to leave as is?

Reply:
I would then change it to
```suggestion
sysCtxResp = map (\x -> x +:+ S "Responsibilities") [titleize user,
  short projectileTitle]
```
to reduce the amount of code duplication